python/podman-compose/Portfile

```diff
@@ -34,16 +34,15 @@ depends_run-append  port:podman \
 subport podman-compose-devel {
     PortGroup           github 1.0
 
-    github.setup        containers podman-compose eeefd37996029ad242edf0b75af34623242b02ea
-    # Change github.tarball_from to 'releases' or 'archive' next update
-    github.tarball_from tarball
+    github.setup        containers podman-compose 342a39dcfe6944932561bf12150deeb687fb6c52
+    github.tarball_from archive
     name                podman-compose-devel
-    version             2024-08-03
+    version             2025-04-14
     revision            0
 
     conflicts           podman-compose
 
-    checksums           rmd160  1662bb579cf0f847619a3a0663aeb3dd9ffc501b \
-                        sha256  87a554a5282ba08e299afbf233a0439f46f031271204e542ff4bc6500b5548a4 \
-                        size    91142
+    checksums           rmd160  96a4a19385e594051c3882c5475f7d107cc5bbca \
+                        sha256  699a09ab5af8eb3dc2e217e28376f34698eaaf1b1cd66c496a74f081db63a8ae \
+                        size    115880
```

Comment:
@BUSTTER the "checksums" for the -devel port appear to be incorrect , at least according to the CI. Please check.

Reply:
ping @BUSTTER